client/checker_client/cli.py

```diff
@@ -0,0 +1,229 @@
+import click
+
+from checker_client import checker as checker_lib
+import pytezos
+from pathlib import Path
+from dataclasses import dataclass
+from marshmallow import Schema, fields
+from marshmallow.decorators import post_load
+import portpicker
+import json
+
+CONFIG_FILE = Path.home().joinpath(Path(".checker"))
```

Comment:
Just to nitpick. We can look at "$XDG_CONFIG_HOME", or default to "$HOME/config" when it is not present. Turns out there is a package called `xdg` which does that, but I don't know if it is worth introducing a new dependency.

Reply:
Done. TIL about $XDG_CONFIG_HOME as well